docs/docs/development/setup/zmk.md

```diff
@@ -0,0 +1,130 @@
+---
+title: ZMK Sources
+sidebar_label: ZMK Sources
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const OsTabs = (props) => (<Tabs
+groupId="operating-systems"
+defaultValue="docker"
+values={[
+{label: 'VS Code & Docker', value: 'docker'},
+{label: 'Debian/Ubuntu', value: 'debian'},
+{label: 'macOS', value: 'mac'},
+{label: 'Windows', value: 'win'},
+{label: 'Raspberry OS', value: 'raspberryos'},
+{label: 'Fedora', value: 'fedora'},
+]
+}>{props.children}</Tabs>);
+
+:::caution Windows Users
+If you're using the Docker environment on Windows, you _must_ checkout the sources to a folder within `C:\Users\[your_user_here]` to avoid a potential permissions issue.
+
+If you're using the WSL2 native filesystem the sources should go under `~/` to avoid potential permissions issues.
```

Comment:
```suggestion
If you're using the WSL2 filesystem the sources should go under `~/` to avoid potential permissions issues.
```

Probably confusing to use "native" here. It may be native to WSL, but everywhere else we're using "native" to refer to not docker and not WSL.